arches_lingo/src/arches_lingo/components/sidenav/SideNav.vue

```diff
@@ -0,0 +1,56 @@
+<script setup lang="ts">
+import { ref } from "vue";
+
+import Button from "primevue/button";
+
+import { routeNames } from "@/arches_lingo/routes.ts";
+
+const items = ref([
+    {
+        icon: "fa fa-home",
+        name: routeNames.root,
+    },
+    {
+        icon: "fa fa-search",
+        name: routeNames.search,
+    },
+]);
+</script>
+
+<template>
+    <aside class="sidenav">
+        <div
+            v-for="item in items"
+            :key="item.name"
+        >
+            <RouterLink :to="{ name: item.name }">
+                <Button>
+                    <i :class="item.icon"></i>
```

Comment:
Let's add an aria-label. Or maybe better, a tooltip? Or a follow-up issue?

Reply:
Done and done

Reply:
Whoops, I guess I shouldn't have suggested one or the other, I think we still need accessibility features on the button/icon combo, probably aria-label on the button and aria-hidden=True on the icon, something like that. I don't think the tooltip is enough here, do you agree @johnatawnclementawn?

Reply:
no worries! Done and done 👍

Reply:
`aria-hidden` on the icon is good, but I'd caution against using a button inside of a link. When I traverse through this using VO, `Home` and `Search` are read out twice first as a link, then as a button. The expected interaction here are navigation to a different page, so best to just use a link there.

Reply:
@johnatawnclementawn this is a good catch!